uint8 MOVE=0
uint8 FORCE_MOVE=1
uint8 SET_ORIGIN=2
uint8 SET_ORIGIN_RHAND=3
uint8 SET_ORIGIN_LHAND=4
uint8 RESET_COORDS=5
uint8 DELETE_FORCE=6
uint8 PUBLISH_MARKER=7
uint8 JOINT_MOVE=8
uint8 RESET_JOINT=9

uint8 SET_MOVE_RARM=10
uint8 SET_MOVE_LARM=11
uint8 SET_MOVE_ARMS=12
uint8 MOVE_CONSTRAINT_T=13
uint8 MOVE_CONSTRAINT_NIL=14
uint8 IK_ROTATION_AXIS_T=15
uint8 IK_ROTATION_AXIS_NIL=16
uint8 USE_TORSO_T=17
uint8 USE_TORSO_NIL=18
int8 USE_FULLBODY=19


uint8 START_GRASP=20
uint8 HARF_GRASP=21
uint8 STOP_GRASP=22

uint8 HEAD_TARGET_POINT=30
uint8 LOOK_AUTO=31

uint8 MANIP_MODE=40
uint8 PICK=41
uint8 TOUCHIT_EXEC=42
uint8 TOUCHIT_PREV=43
uint8 TOUCHIT_CANCEL=44

uint8 LOOK_RARM=45
uint8 LOOK_LARM=46

uint8 PLAN=50
uint8 EXECUTE=51
uint8 PLAN_EXECUTE=52
uint8 CANCEL_PLAN=53

uint8 GENERAL=0
uint8 HEAD_MARKER=1
uint8 RHAND_MARKER=2
uint8 LHAND_MARKER=3
uint8 RLEG_MARKER=4
uint8 LLEG_MARKER=5
uint8 BASE_MARKER=6
uint8 RFINGER_MARKER=7
uint8 LFINGER_MARKER=8

int8 menu
int8 type
string marker_name